packages/design-tokens/src/MargaritaTokens.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+export const margaritaTokens = {
+  widthScreenNormal: 720,
+  textColorBlueDark: '#2e353b',
+  borderColorGreyLight: '#d2d9e0',
```

Comment:
In terms of color names, I would stick to orbit's. We don't use `grey`, but `cloud` and `ink` instead. Both of these colors are ink-ish.

Reply:
Thanks, updated.